### ISSUE TITLE
Implement source and quality monitoring

### ISSUE THREAD

**Issue by jzohrab** [covidatlas/li#234]
## Description.

Currently, we don't know if sources are actually working.  Source URLs can disappear or change, and we currently don't have an idea what's working and what's not.

This can be further broken down into two types of monitoring/reporting:

* Data source check: Each data source (URL) may be missing or present, and the data either scraped successfully or not.
* Data quality check: a given data source may exist and be scraped, but the results may be nonsense.  We can likely catch some of these issues.

### Data source checks

This could be a simple page listing sources, and their status:

`| source | status (up || down) | last successful crawl | last successful scrape | error message (if status = down) |`

### Data quality checks

Some examples of data quality issues that we can likely monitor and catch early:

https://docs.google.com/document/d/1vwW6XiCGpQPbhMxMhLISwAPbFemr4UIBV8qIqVxdJBM/edit#

These can be broken down into separate issues, but two that would be useful to implement soon would be "Missing Data" and "Zero Data".  We would need to have timeseries generation for sources for some of these checks.

**Comment by ryanblock** [comment]
This looks solid and I think accurately describes where the `status` system is going!

**Comment by jzohrab** [comment]
Opened https://github.com/covidatlas/li/pull/238 to address the "data source checks".  "Data quality checks" belongs in its own issue, will move it there once #238 is merged.

**Comment by jzohrab** [comment]
Source checks implemented, https://api.staging.covidatlas.com/status?format=html is up.  :tada:
